com.vagueentertainment.Framed.yml

```diff
@@ -0,0 +1,81 @@
+app-id: com.vagueentertainment.Framed
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-nightly
```

Comment:
I don't see it being used anywhere?

Reply:
I think I can remove this, there was a python module that was complaining about not having rust installed. But I think I found a work around for that.